project/src/components/place-card-component/place-card-component.tsx

```diff
@@ -0,0 +1,40 @@
+function PlaceCard(): JSX.Element {
+  return (
+    <article className="cities__place-card place-card">
+      <div className="place-card__mark">
+        <span>Premium</span>
+      </div>
+      <div className="cities__image-wrapper place-card__image-wrapper">
+        <a href="/">
+          <img className="place-card__image" src="img/apartment-01.jpg" width="260" height="200" alt="Place" />
```

Comment:
посмотри, и там где в компоненты передается много пропсов, лучше делать вот так

```suggestion
          <img 
            className="place-card__image" 
            src="img/apartment-01.jpg" 
            width="260" 
            height="200"
            alt="Place" 
          />
```

Reply:
ок